src/functions/leaderboard/leaderboard.go

```diff
@@ -0,0 +1,27 @@
+package leaderboard
+
+import (
+	"strings"
+	"encoding/json"
+	"net/http"
+)
+
+const BASE_URL = "https://archybot.web.app/leaderboard/"
+
+var Payload struct {
+	ServerId  string `json:"server_id"`
+}
+
+func sendLeaderboardUrl(w http.ResponseWriter, r *http.Request){
```

Comment:
Public function name must start with an UPPER_CASE
`sendLeaderboardUrl` -> `SendLeaderboardUrl`

Reply:
Alright, I will correct it